mkdocs.yml

```diff
@@ -5,11 +5,24 @@ repo_name: alandefreitas/small
 
 site_description: "Small: C++ Small Containers"
 
+nav:
+  - Home: index.md
+  #  - Quickstart: quickstart.md todo
+  #  - Motivation: motivation.md todo
+  - Reference: # Reference is organized by module
+      - Small: reference/Modules/index.md        # futures module
```

Comment:
We have to figure out the best reference structure for this library. 

We usually have one module per link in the sidebar, but this library is too small for that. We don't even have oxygen modules defined.

We could create one module/link per container or just a single reference link with all the containers and functions. One of the reference indexes should do for now.

Reply:
And btw, we need `mkdocs serve --dirtyreload` to play with the docs and find the best structure.

Reply:
I'm sorry for the delayed response. Do you think we need to change the sidebar references?

Reply:
This is resolved, right? I imagine you already got this.